Impossible-Agent-Suite/non-whitelist-mint/README.md

```diff
@@ -0,0 +1,36 @@
+# Impossible Finance token non-whitelist mint
+
+## Description
+
+This agent detects IF and IDIA token mints to non-whitelisted addresses, excluding mints through the function `staxMigrate`
+
+## Supported Chains
+
+- Ethereum
+- Binance Smart Chain
+
```

Comment:
based on docs, Polygon is a supported chain as well

Reply:
Same situation as [here](https://github.com/NethermindEth/Forta-Agents/pull/154#discussion_r811611087)